### DISCUSSION TITLE
how to keep pruning the chain store,it grows so fast !

### DISCUSSION THREAD

**Discussion by KIMQI001** [filecoin-project/lotus#5665] [Storage Provider] [answered]
the chain store grows so fast,my disk is going to be full~

**Comment by Meatball13** [comment, marked as answer]
Yeah, the chain grows pretty fast, you can compact the chain and pull down a pruned chain though to get it back under control.  I've been doing it every 4-6 weeks.  Link below has directions on how to do it, just be prepared that it's going to take a few hours at least for it to download the new chain and the daemon will be down during that time, so do it as part of maintenance or something.

https://docs.filecoin.io/get-started/lotus/chain/#compacting-the-chain-data

**Reply by KIMQI001** [reply]
thx, I saw the 'lotus-shed state-prune' also make sense,but it pruned so inefficiently.Now I want to do something let the daemon prune automaticly.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
The snapshots can be downloaded in advance while the old chain is still running. Then import the snapshot from the local disk
